apps/aecore/lib/aecore/tx/signed_tx.ex

```diff
@@ -77,8 +82,7 @@ defmodule Aecore.Tx.SignedTx do
      - tx: The transaction data that it's going to be signed
      - priv_key: The priv key to sign with
   """
-  @spec sign_tx(DataTx.t() | SignedTx.t(), binary()) ::
-          {:ok, SignedTx.t()} | {:error, String.t()}
+  @spec sign_tx(DataTx.t() | SignedTx.t(), binary()) :: {:ok, SignedTx.t()} | {:error, String.t()}
```

Comment:
{:error, String.t()} is repeated a lot. Maybe define a error() type?

Reply:
I think I would leave such changes to some refactoring later as this applies to most of our codebase. I'm open to introducing such changes to channels in channels PR, but we shouldn't refactor whole codebase in channels PR.